build/config/config.js

```diff
@@ -0,0 +1,9 @@
+// import dotenv from 'dotenv'
+// dotenv.config();
+// const env = process.env.NODE_ENV;
```

Comment:
Hey man, I think it will be better if you remove the commented lines of code.

Reply:
Thank you, let's focus on that!